cypress/integration/register_device_page_test.js

```diff
@@ -0,0 +1,54 @@
+import { urlSafeBase64ToByteArray } from '../../src/react/Base64.ts';
+
+describe('Register device page tests', () => {
+  context('no access before login', () => {
+    it('redirects to home', () => {
+      cy.visit('/devices/register');
+      cy.location('pathname').should('eq', '/login');
+    });
+  });
+
+  context('require login', () => {
```

Comment:
`require login` makes me believe the next tests will check if the user is requested to login.
Could something like `authenticated` be a better phrasing for the context?

Reply:
that's a good point!